Source/Engine/Platform/Windows/WindowsClipboard.cpp

```diff
@@ -25,7 +25,7 @@ void WindowsClipboard::SetText(const StringView& text)
 {
     const int32 size = (text.Length() + 1) * sizeof(Char);
     const HGLOBAL hMem = GlobalAlloc(GMEM_MOVEABLE, size);
-    Platform::MemoryCopy(GlobalLock(hMem), text.GetText(), size);
+    Platform::MemoryCopy(GlobalLock(hMem), String(text).GetText(), size);
```

Comment:
If text to copy is large then allocating it into memory might be slower. Maybe we could do separate copy for text and manually insert null-terminator in the end?

Reply:
Improved as suggested.